package.json

```diff
@@ -18,7 +18,8 @@
     "watch": "npm run build:dev -- --watch",
     "release-zip": "npm run build && npm run zip",
     "test": "npm run build:production -- --display=none && bundlesize",
-    "test:visualtest": "npm run build:static && start-server-and-test storybook:vrt http-get://localhost:9001 backstopjs",
+    "pretest:visualtest": "npm run build:storybook --silent",
+    "test:visualtest": "npm run backstopjs -- --storybook-host=./dist/",
```

Comment:
Passing `--storybook-host` here results in the argument being set twice in the `config.js`

```
[ '/usr/local/bin/node',
  '/usr/local/bin/backstop',
  'test',
  '--config=tests/backstop/config.js',
  '--moby',
  '--storybook-host=./dist/',
  '--storybook-host=http://host.docker.internal:9001/' ]
```

Reply:
Updated.